Scalar.FunctionalTests/Tests/MultiEnlistmentTests/ScalarCloneFromGithub.cs

```diff
@@ -0,0 +1,75 @@
+﻿using NUnit.Framework;
+using Scalar.FunctionalTests.FileSystemRunners;
+using Scalar.FunctionalTests.Tools;
+using Scalar.Tests.Should;
+using System;
+using System.IO;
+using System.Linq;
+
+namespace Scalar.FunctionalTests.Tests.MultiEnlistmentTests
+{
+    [Category(Categories.GitRepository)]
+    public class ScalarCloneFromGithub : TestsWithMultiEnlistment
+    {
+        private static readonly string MicrosoftScalarHttp = "https://github.com/microsoft/scalar";
+        private static readonly string MicrosoftScalarSsh = "git@github.com:microsoft/scalar.git";
+
+        private FileSystemRunner fileSystem;
+
+        public ScalarCloneFromGithub()
+        {
+            this.fileSystem = new SystemIORunner();
+        }
+
+        [TestCase]
+        public void PartialCloneHttps()
```

Comment:
I dropped the SSH test because I couldn't get `known_hosts` to work. It's still in the history of this branch.